src/views/report/ScriptReportsOverview/index.tsx

```diff
@@ -182,9 +183,14 @@ const ScriptReportsOverview = withStyles(styles)(
         public componentDidMount() {
             const { dispatch } = this.props;
             const initialFilters = this.combineFiltersFromUrlAndCurrentFilters();
+            const timeToRefresh = configData.iesi_time_to_refresh_in_seconds * 1000;
 
             this.fetchExecutionRequestsWithFilterAndPagination({ newListFilters: initialFilters, newPage: 1 });
             dispatch(setExecutionsListFilter({ filters: initialFilters }));
+            setInterval(() => {
+                this.fetchExecutionRequestsWithFilterAndPagination({ newListFilters: initialFilters, newPage: 1 });
```

Comment:
If an user if in the third page, do you want to display the first page after x seconds ? Fix it